packages/front-end/components/EventWebHooks/EventWebHookAddEditModal/EventWebHookAddEditModal.tsx

```diff
@@ -49,6 +67,8 @@ export const EventWebHookAddEditModal: FC<EventWebHookAddEditModalProps> = ({
       close={onClose}
```

Comment:
`cta` should be "Create" | "Edit" based on context

Reply:
I've implemented that suggestion (Save vs Create) here abc1a5e64e6d6f2e5bdbf7aa07b94cb9c2fe532a